src/QuadForm/IO.jl

```diff
@@ -23,24 +23,24 @@ function to_hecke(io::IO, L::QuadLat; target = "L", skip_field = false)
     pol = string(f)
     pol = replace(pol, string(var(parent(f))) => "x")
     println(io, "f = ", pol, ";")
-    println(io, "K, a = number_field(f)")
+    println(io, "K, a = number_field(f, \"a\", cached = false)")
```

Comment:
why change to `cached = false`?

Reply:
I wanted to make the functions for quadratic and hermitian lattices consistent regarding each other, so I added this here as it is done for hermitian lattices